src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1082,7 +1360,8 @@ private static GencodeGtfFeature getContainingGtfSubfeature(final VariantContext
             // where this overlaps something more meaningful.
             // For example, see HG19 - chr19:8959608
             for (final GencodeGtfExonFeature exon : transcript.getExons()) {
-                if ((exon.getCds() != null) && (exon.getCds().overlaps(variant))) {
+                // TODO: It would be really nice if you could make this work for the OVERLAPS method not just the CONTAINS method.  This is in here for issue #4307.
```

Comment:
I don't understand this todo.

Reply:
Part of that `todo` was leftover from another note I had previously.  I've updated it with a better explanation.